### PR TITLE
feat: Allow to modify prefix and keyboard name

### PR DESCRIPTION
**Pull request by ermshiperete**
This change also implements #68 (Check if repo contains single keyboard).

### CODE REVIEW

File: frontend/src/app/components/pull-request/pull-request.component.html [hunk 1]
```diff
@@ -8,10 +8,19 @@
...
+  </div>
+  <div *ngIf="!pullRequest && !message">
     <div class="spinner">
       <fa-icon icon="spinner" spin="true" size="5x"></fa-icon>
     </div>
-    <div class="message">Please wait while we create the pull request on your behalf...</div>
+    <div class="message">Please wait while we
+      <span *ngIf="updatePullRequest">update</span>
+      <span *ngIf="!updatePullRequest">create</span> the pull request on your behalf...
+    </div>
```
Comment (mcdurdin) [inline frontend/src/app/components/pull-request/pull-request.component.html:24]: For future i18n, easier if we don't do too much grammatical assembly.

```suggestion
    <div class="message" *ngIf="updatePullRequest">Please wait while we update the pull request on your behalf...</div>
    <div class="message" *ngIf="!updatePullRequest">Please wait while we create the pull request on your behalf...</div>
```

File: src/pull-request/pull-request.service.ts [hunk 2]
```diff
@@ -146,19 +156,49 @@ export class PullRequestService {
...
+  public getPrefix(keyboardId: string, prefix: string): string {
+    if (!keyboardId) {
+      return keyboardId;
+    }
+
+    if (prefix) {
+      return prefix;
+    }
+
+    const specialPrefixes = [
```
Comment (mcdurdin) [inline src/pull-request/pull-request.service.ts:184]: This will need to be built from the keyboards repo folder names as it does change frequently.

Comment (ermshiperete) [reply src/pull-request/pull-request.service.ts:184]: https://github.com/keymanapp/developer.keyman.com/issues/73

File: frontend/src/app/components/pull-request/pull-request.component.html (hunk 1 above)
Comment (ermshiperete) [reply frontend/src/app/components/pull-request/pull-request.component.html]: Good idea. Makes it easier to read.

**Pull request merged**